SwiftWisdomTests/StandardLibrary/Numbers/UnsignedInteger+ExtensionsTests.swift

```diff
@@ -19,4 +19,207 @@ class UnsignedIntegerExtensionTests: XCTestCase {
         let mixed = -100...100
         XCTAssert(mixed.contains(random(inRange: mixed)))
     }
+
+    func testRandom() {
+        //XCTAssert()
+        //XCTAssertFalse()
+    }
+    
+    //Finished
+    func test_ip_initData() {
+        var data = Data(byte: UInt8(0))
+        var number = UInt8.init(ip_data: data)
+        XCTAssert(number == UInt8(0))
```

Comment:
This, and all similar tests throughout the PR, should be calls to `XCTAssertEqual`